examples/tabset/app.R

```diff
@@ -2,28 +2,47 @@ library(shiny)
 library(shiny.semantic)
 
 ui <- shinyUI(semanticPage(
+  action_button("changetab", "Select First Tab!"),
+  br(),br(),
+  uiOutput("activetab"),
+  br(),br(),
+  selectInput(
+    inputId = "plot1xaxis",
+    label = "change x axis variable in third tab:",
+    choices = names(mtcars),
+    selected = "mpg",
+    width = 200
+  ),
+  br(),br(),
   tabset(tabs =
            list(
              list(menu = "First Tab", content = "Text works well", id= "first_tab"),
              list(menu = "Second Tab", content = plotOutput("plot1"), id = "second_tab"),
-             list(menu = "Third Tab", content = plotOutput("plot2"))
+             list(menu = "Third Tab", content = plotOutput("plot2"), id = "third_tab")
```

Comment:
could you remove the id here? just to show the user that this is not a mandatory field